### ISSUE TITLE
A11y_Code.visualstudio.com_Visual studio API_Top navigation bar_High contrast: In high contrast blank and white mode, it will be more user friendly, if the focus border for "dismiss this update" button is provided as a dotted border/in different color.

### ISSUE THREAD

**Issue by MukallaDivya** [microsoft/vscode-docs#5185]
## GitHub Tags:
#A11yTCS;#Win10;#WCAG4.3.1;#DesktopWeb;#A11ySev4;#ChromiumEdge;#P1-VisualStudioCodeServices-Web-Feb2022;#P1_Visual Studio Code Services;#Visual Studio Code Services;#A11yUsable;
## Environmental Details:
https://code.visualstudio.com/api
## OS & Browser Details :
Windows 10 Enterprise 21H2(OS Build 19044,1466)
Microsoft Edge Version 98.0.1108.43 (Official build) (64-bit)
## Pre-Requisite:
To apply high contrast follow the below steps:
1. Open settings, tab till Ease of access and press enter.
2. Tab till High contrast press enter.
3.  From the drop down, select a proper theme by pressing enter. Use Black and white themes.
4. Tab till apply button and press enter.
## Repro Steps :
1. Launch the URL https://code.visualstudio.com/api
2. Turn on High contrast in settings.
3. Tab till "dismiss the update button" and verify the focus visibility is proper or not .

### Note issue :
This issue is also observed in this URL  https://code.visualstudio.com/learn .

## Actual Result :
 In high contrast blank and white mode, the focus border for "dismiss this update"  button is not  provided with a dotted border/in different color.
## Expected Result:
In high contrast blank and white mode, it will be more user friendly, if the focus border for "dismiss this update" button is provided as a dotted border/in different color. 
## User Impact :
Users not able to understand clearly and got confuse, when that "dismiss the update" button received  the focus.
## Attachments :

https://user-images.githubusercontent.com/99255594/154278369-2b3efb38-ef01-40ee-9085-f9a680c90d26.mp4

https://user-images.githubusercontent.com/99255594/154278471-495e53a3-43f6-4af0-a76c-e8d352984830.mp4

**Comment by gregvanl** [comment]
Won't fix. The site is currently highlighting the "Dismiss the update" square in high contrast.
Using a dotted line border would not match the other highlights used on the site and not sure changing the color is required.
